docs/contracts/errors.rst

```diff
@@ -40,7 +40,7 @@ as well as the newer approach with ``require`` in function ``transferWithRequire
             balance[to] += amount;
         }
         function transferWithRequireError(address to, uint256 amount) public {
-            require(amount > balance[msg.sender], InsufficientBalance(balance[msg.sender], amount));
+            require(amount < balance[msg.sender], InsufficientBalance(balance[msg.sender], amount));
```

Comment:
Shouldn't it be `<=`?
```suggestion
            require(amount <= balance[msg.sender], InsufficientBalance(balance[msg.sender], amount));
```

Reply:
Uh, yeah, it should be `<=` :)

Reply:
It should, but then the `if (amount > balance[msg.sender]) revert ...` needs to be updated to match too. Otherwise they're no longer eqivalent.

Reply:
> It should, but then the `if (amount > balance[msg.sender]) revert ...` bit above needs to be updated to match too. Otherwise they're no longer eqivalent.

They are equivalent though.

Reply:
Ah, true. My bad :)